src/styles/components/_modal.scss

```diff
@@ -42,7 +41,6 @@ $component-name: modal;
       flex-direction: column;
       align-items: center;
       justify-content: center;
-      max-height: 20vh;
```

Comment:
I removed these max-heights from both versions; overflow is visible, so if the title is that long, it obscures form text anyway. We could limit to just the fullscreen version, however.